recipe/meta.yaml

```diff
@@ -10,19 +10,20 @@ source:
     sha256: 171dd39dbcd814995f59eec5a23646cae12c08e5ef2e0e9c986198e220d2d038
 
 build:
-    number: 1
+    number: 2
     skip: true  # [win]
- 
+
 requirements:
     build:
         - {{ compiler("c") }}
     host:
-        - python {{ python }}
+        - python
+        - cython
```

Comment:
@muryanto1, I think the lack of cython as a dependency might be the main build problem here (it might be picking up an incompatible system version).

The python pinning is sometimes also problematic, but I don't know if it's related to the build problems with python 3.7.6.